data/extension/init/templates/tinygo/.gitignore

```diff
@@ -0,0 +1,2 @@
+# We rename go.mod to go.mod_ to workaround https://github.com/golang/go/issues/45197
```

Comment:
@mathetake this will only effect tinygo as I don't expect any other extension language to want a file named go.mod in it.